beta-src/src/state/game/game-api-slice.ts

```diff
@@ -164,28 +164,31 @@ const highlightMapTerritoriesBasedOnStatuses = (
   filter: Territory[] = [],
 ) => {
   const {
-    data: { data: gameData },
+    territoriesMeta,
     overview: { members },
+  }: {
+    territoriesMeta: TerritoriesMeta;
+    overview: {
+      members: GameOverviewResponse["members"];
+    };
   } = current(state);
-  if ("territories" in gameData && gameData.territories) {
+  if (Object.keys(territoriesMeta).length) {
     const membersMap = {};
     members.forEach((member) => {
       membersMap[member.countryID] = member.country;
     });
-    const t: ITerritory[] = Object.values(gameData.territories);
-    t.forEach(({ countryID, name }) => {
-      const country = membersMap[countryID];
-      const mappedTerritory = TerritoryMap[name];
-      if (filter.length && !filter.includes(mappedTerritory.territory)) {
+    Object.values(territoriesMeta).forEach((terr: TerritoryMeta) => {
+      const { ownerCountryID, territory } = terr;
+      const country = ownerCountryID ? membersMap[ownerCountryID] : undefined;
+      if (filter.length && !filter.includes(territory)) {
         return;
       }
-      const terrEnum = Territory[mappedTerritory.territory];
-      if (terrEnum) {
+      if (territory) {
```

Comment:
You're checking for filter.includes(territory) on L183. If the check above passes we don't nee to check again.

Reply:
That is a diff check meant to only highlight specific territories, however, I will remove thatt since it isn't even being used anyways.